packages/api/src/router/chats.ts

```diff
@@ -1,3 +1,4 @@
+import { prisma } from "@oakai/db/client";
```

Comment:
@codeincontext if we don't want to change the global import behaviour, we need to add this anyway. This file isn't importing prisma at the moment because of the declare global in db/client.

I'm not sure how this isn't throwing an error.

Reply:
Yeah, very odd. Maybe we don't actually call these endpoints? I've approved #342